web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -1,4 +1,8 @@
 <template>
+  <!--
+    Do we want to set the width here? Default is nearly the same.
+    If we do want to set wouldn't width=500 be sufficient?
+  -->
```

Comment:
```suggestion
```

Reply:
Definitely will be removing the comment, just looking to bring attention to this.

It is odd to have a set width instead of something responsive. It is even odder to have a set width in this manner: 
```
min-width:500px;
max-width:500px;
```

I am bringing attention to this now because the DownloadDialog is the component we chose to influence the design of the ContactDialog

Reply:
comment removed in 55db5cd4b43e05d463bc64f895b523248b54b653